feat(components/notifications): add documentation for global alert (#446)
zendeskgarden/website: src/pages/components/global-alert.mdx

      <li>Announce new versions of products</li>
      <li>Inform users of trial status</li>
      <li>Let users know they are in an editing mode</li>
      <li>Communicate system maintenance or disconnection from servers</li>
    </ul>
  </Use>
  <Misuse>
    <ul>
      <li>
        <Markdown>
          For more contextual feedback about an action or task, use an
          [Alert](/components/notifications)
        </Markdown>
      </li>
    </ul>
  </Misuse>
</Usage>

## How to use it

### Default

import Default from '../../examples/notifications/global-alert/GlobalAlertDefault.tsx';
import DefaultCode from '!!raw-loader!../../examples/notifications/global-alert/GlobalAlertDefault.tsx';

<CodeExample code={DefaultCode}>
  <Default />
</CodeExample>

### Type

Types indicate what kind of message is in the Global alert and help the user understand
how to respond. They can be info, warning, error, or success.

import Types from '../../examples/notifications/global-alert/GlobalAlertTypes.tsx';
import TypesCode from '!!raw-loader!../../examples/notifications/global-alert/GlobalAlertTypes.tsx';

<CodeExample code={TypesCode}>
  <Types />
</CodeExample>

## How to use it well

> ### Clearly actionable

Reviewer: This section should have an associated example, as it is imperative for developers to see how to properly use anchors.

Author: @jzempel agreed - FYI, we added a variant with a link example.

Allow for one action button for the user to address the issue

> associated with the banner, with the exception of “learn more”.

Author: @jzempel yeah for sure, could you give a restructured example?

> associated with the banner, with the exception of “learn more”.

Reviewer: We can use https://hemingwayapp.com/ to button this up. But I'm having a difficult time understanding what it actually means.

> Allow for one action button for the user to address the issue
> associated with the banner, with the exception of “learn more”.

Author: ```suggestion
Provide one button to address the issue, and an optional way to “learn more”.
```
